### DISCUSSION TITLE
Bedrock, output of Llama 3.1 405B model just stops

### DISCUSSION THREAD

**Discussion by dirkpetersen** [danny-avila/LibreChat#4130] [Troubleshooting]
### What happened?

When I ask the Llama 3.1 405B  model on Bedrock to return a page, it will just stop after a while without completing. This does not happen with the Llama 3.1 70B model or any other models. It also does not happen when I query bedrock on the CLI or via Python. My current streamRate is 75 but i slowed it down to as much as 250 with no effect 

### Steps to Reproduce

- Select Bedrock
- Select meta.llama3-1-405b-instruct-v1:0
- Enter Prompt, e.g. "Give me a full page on little known facts about Mark Twain"

### What browsers are you seeing the problem on?

Firefox

### Relevant log output

```shell
No log output
```

### Screenshots

![image](https://github.com/user-attachments/assets/398953bf-e5db-45bf-a4f2-45555b30081a)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
The same is observed on Amazon Bedrock Chat playground:
![image](https://github.com/user-attachments/assets/2b167b74-1a80-46dd-9c83-d2fb26f383b5)

It seems that response length is lower for 405b than the other llama models

![image](https://github.com/user-attachments/assets/6a5527aa-180a-4dd6-ad2f-9bcfce8c63c4)

70b:

![image](https://github.com/user-attachments/assets/78e76407-fc75-4de4-ab60-7042cf2bf4d1)

I will have to add an inference parameter for llama models for this

**Comment by dirkpetersen** [comment]
ah that makes sense ..... for my testing with Python I had already globally increased that length
